### DISCUSSION TITLE
Asus fan extension card

### DISCUSSION THREAD

**Discussion by parapixology** [Rem0o/FanControl.Releases#1064] [Q&A]
Hello!

Is there a way to have this app work with the Asus fan extension card?

Thanks,
Daniel

**Comment by Rem0o** [comment]
There is no open API for me to access it AFAIK.

**Reply by parapixology** [reply]
Thanks for the reply. That is a bummer, I really like the app. I use it on my secondary machine. Do you know of an alternate fan card I can use  that will work with Fan Control?

Thanks!
D.

**Reply by 1ccs-todd** [reply]
i got FanControl to see the fan card temp & fan speeds via hwinfo64 plugin.
just cannot control them.
weird thing is I am able to control W_PUMP+ header once I added the rpm sensor via same hwinfo plugin.
